### PR TITLE
Re-use send(Info/Error)Message for interaction responses

### PR DESCRIPTION
**Pull request by Brainicism**

### CODE REVIEW

File: src/helpers/discord_utils.ts [hunk 1]
```diff
@@ -376,13 +376,29 @@ async function sendMessageExceptionHandler(
...
+    authorID?: string,
+    interaction?: Eris.ComponentInteraction | Eris.CommandInteraction
 ): Promise<Eris.Message> {
+    if (interaction) {
+        if (!withinInteractionInterval(interaction)) {
+            return null;
+        }
+
+        try {
+            await interaction.createMessage(messageContent);
```
Comment (taahamahdi) [inline src/helpers/discord_utils.ts:394]: should we return the message?

**Review by taahamahdi — CHANGES_REQUESTED** [review]

File: src/helpers/discord_utils.ts (hunk 1 above)
Comment (Brainicism) [reply src/helpers/discord_utils.ts:394]: it returns Promise<void> and we don't ever use the return value

Comment (taahamahdi) [reply src/helpers/discord_utils.ts:394]: This function returns `Promise<Eris.Message>`

Comment (Brainicism) [reply src/helpers/discord_utils.ts:394]: https://abal.moe/Eris/docs/0.17.1/CommandInteraction#method-createMessage
image
image

Comment (taahamahdi) [reply src/helpers/discord_utils.ts:394]: Whoops, my bad

**Review by taahamahdi — APPROVED** [review]

**Pull request merged**